### ISSUE TITLE
 Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]

### ISSUE THREAD

**Issue by ArturiaPendragon** [debakarr/kodekloud-downloader#55]
If in run `kodekloud dl -c learn.kodekloud.com_cookies.txt -o . ` and i choose '2' or '1-103' i have this error :

`Traceback (most recent call last):
  File "/home/arturia/.local/bin/kodekloud", line 8, in <module>
    sys.exit(kodekloud())
             ^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arturia/.local/lib/python3.12/site-packages/kodekloud_downloader/cli.py", line 69, in dl
    download_course(
  File "/home/arturia/.local/lib/python3.12/site-packages/kodekloud_downloader/main.py", line 126, in download_course
    fetch_course_detail(course.slug) if isinstance(course, Course) else course
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arturia/.local/lib/python3.12/site-packages/kodekloud_downloader/models/helper.py", line 36, in fetch_course_detail
    return CourseDetail(**data)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/arturia/.local/lib/python3.12/site-packages/pydantic/main.py", line 209, in __init__
    validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.ValidationError: 1 validation error for CourseDetail
excerpt
  Input should be a valid string [type=string_type, input_value=None, input_type=NoneType]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type`

**Comment by debakarr** [comment]
Thanks for reporting. This has a quick fix. For now you can try this command to install changes from a feature branch until I merge it to main.

```console
pip install -U git+https://github.com/debakarr/kodekloud-downloader.git@handle-empty-excerpt --force-reinstall
```

**Comment by debakarr** [comment]
Fixed in #57
